mlir/include/mlir/IR/BlockSupport.h

```diff
@@ -74,8 +74,8 @@ class SuccessorRange final
 public:
   using RangeBaseT::RangeBaseT;
   SuccessorRange();
-  SuccessorRange(Block *block);
-  SuccessorRange(Operation *term);
+  SuccessorRange(Block *block LLVM_LIFETIME_BOUND);
```

Comment:
What's the value of adding it here? The address of these aren't taken.

Reply:
Thanks River for noticing! Removed.